server/cloudformation/cloudformation.go

```diff
@@ -86,11 +86,11 @@ type Request struct {
 	// This field contains the contents of the Properties object sent by the
 	// template developer. Its contents are defined by the custom resource
 	// provider.
-	ResourceProperties interface{} `json:"ResourceProperties"`
+	ResourceProperties map[string]interface{} `json:"ResourceProperties"`
```

Comment:
Maybe easier to make this a json.RawMessage so resource providers can just decode into a struct.